components/admin/admin-touch-interactions.tsx

```
@@ -0,0 +1,342 @@
+      className={cn(
+        "inline-flex items-center justify-center rounded-lg font-medium",
+        "transition-all duration-200 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2",
+        TOUCH_TARGET_SIZE,
+        TOUCH_FEEDBACK_STYLES,
+        getVariantStyles(),
+        getSizeStyles(),
+        fullWidth ? "w-full" : "",
+        disabled ? "opacity-50 cursor-not-allowed" : "cursor-pointer",
+        className
+      )}
+    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)